source/templates/displaying-a-list-of-items.md

```diff
@@ -1,4 +1,11 @@
-If you need to enumerate over a list of objects, use Handlebars' `{{#each}}` helper:
+To iterate over a list of items, use the `{{#each}}` helper. The first
+argument to this helper is the array to be iterated, and the value being
+iterated is yielded as a block param. Block params are only available inside
+the block of their helper.
+
+For example, this template iterates an array named `people` that contains
+objects. Each item in the array
+is yielded as a block param named `person` by the template.
```

Comment:
I like how you said it below - "with the each item set to the `person` block param".